tests/integration/interface_common_test.py

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
+#
+# This file is part of nmstate
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+import libnmstate
+
+from libnmstate.schema import Interface
+
+
+def test_iface_description_removal(eth1_up):
+    desired_state = eth1_up
+    eth1_desired = next(
+        ifstate
+        for ifstate in desired_state[Interface.KEY]
+        if ifstate[Interface.NAME] == 'eth1'
+    )
+    eth1_desired[Interface.DESCRIPTION] = 'bar'
+    libnmstate.apply(desired_state)
+    current_state = libnmstate.show()
+    eth1_current = next(
+        ifstate
+        for ifstate in current_state[Interface.KEY]
+        if ifstate[Interface.NAME] == 'eth1'
+    )
```

Comment:
Please use `show_only`: https://github.com/nmstate/nmstate/blob/5701bbe97e23e66f8966993fbe60b99e1796168b/tests/integration/testlib/statelib.py#L41

Then you can access it directly (as there is only one interface).